extensions/firefox/content/PdfjsContentUtils.jsm

```diff
@@ -17,12 +17,6 @@
 
 var EXPORTED_SYMBOLS = ["PdfjsContentUtils"];
 
-const Cc = Components.classes;
-const Ci = Components.interfaces;
-const Cr = Components.results;
-const Cu = Components.utils;
-
-ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
```

Comment:
Is this line removal intended? I don't see it being removed in the Bugzilla patch or on the other linked pages.

Reply:
Sorry, I probably shouldn't have lumped this in while upstreaming the other changes.
However this has been unused ever since PR #5317, and we'll probably need to remove it soon anyway; see https://bugzilla.mozilla.org/show_bug.cgi?id=1436961.